_posts/2020-06-21-one-stop-guide-to-lombok.md

```diff
@@ -0,0 +1,337 @@
+---
+title: "One-stop Guide to Lombok"
+date: 2020-06-21 10:00 +0500
+modified: 2020-06-21 10:15 +0500
+---
+
+# What is it
+Lombok is a Java library which makes Java less chatty whilst making your code more readable. Java is great language but when you use its features you have to write a lot of boilerplate code.
+
+# Why to use Lombok
+
+Lombok helps you with a set of annotations to make your code more readable without losing its functionality. You don’t need to write typical constructors, setters or getters. With the help of Lombok you can easily add logging feature to you class or create fully functional builder.
+
+# Which the most popular features it has
```

Comment:
This isn't valid english. I usually fix small language problems while going through the article, but there are too many in this one. Please run the text through Grammarly.